src/mongod-helper.ts

```diff
@@ -76,6 +77,10 @@ export class MongodHelper {
     return /waiting for connections on port/i;
   }
 
+  getMongodStartedExpression2(): RegExp {
+    return /\[initandlisten\] setting featureCompatibilityVersion/i;
+  }
```

Comment:
Couldn't we use a single regexp with the two expressions, rather than two different functions ?
Something like `/(waiting for connections on port|\[initandlisten\] setting featureCompatibilityVersion)/i`

Reply:
Due to the performance it has no significant impact but it's more readable. The variable names could be renamed.

Reply:
I agree, parsing logs with regexp is pretty bad performance-wise anyway. And `getMongodStartedExpression2` is not a very descriptive function name, I only suggested that for readability.

Reply:
Yes,  this should be renamed.